### ISSUE TITLE
[FEATURE] Expose CardIO hideCardIOLogo property

### ISSUE THREAD

**Issue by isimpson** [verygoodsecurity/vgs-collect-ios#159]
CardIO library provides the opportunity to hide the PayPal logo on the card scanner but the property is not exposed through the `VGSCardIOScanController`.

See:
```
/// Hide the PayPal or card.io logo in the scan view. Defaults to NO.
@property(nonatomic, assign, readwrite) BOOL hideCardIOLogo;
```

**Comment by dmytrokhl** [comment]
Hello @isimpson  Yes, I think it's better we hide the logo by default. We will add the update in next release.

**Comment by dmytrokhl** [comment]
This is released in [v1.5.4](https://github.com/verygoodsecurity/vgs-collect-ios/releases/tag/1.5.4).
